parse_font.mayhemfile

```diff
@@ -0,0 +1,7 @@
+project: rusttype
```

Comment:
Tests has been changed to testsuite.

Reply:
resolved by bundling it in the docker image